### PR TITLE
refactor: introduce NullAction

### PR DESCRIPTION
**Pull request by kfangw**
Introduce `NullAction` for the usage of irrelevant to concrete action type.

### CODE REVIEW

File: Libplanet/Store/StoreExtensions.cs [hunk 1]
```diff
@@ -58,27 +58,5 @@ public static void Copy(this IStore from, IStore to)
                 to.SetCanonicalChainId(canonId);
             }
         }
-
```
Comment (longfin) [inline Libplanet/Store/StoreExtensions.cs:61]: Can we delete `using Bencodex.Types` too?

Comment (kfangw) [reply Libplanet/Store/StoreExtensions.cs:61]: https://github.com/planetarium/libplanet/pull/1387/commits/7b2feba8d47ca2f6a5bd82c68abff466ef7f9eda

**Review by longfin — APPROVED** [review]

File: Libplanet/Action/NullAction.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+using Bencodex.Types;
```
Comment (dahlia) [inline Libplanet/Action/NullAction.cs:1]: All newly added files should turn on the null checker.  See also <https://github.com/planetarium/libplanet/issues/458#issuecomment-669674809> for details.

```suggestion
#nullable enable
using Bencodex.Types;
```

File: Libplanet/Action/NullAction.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+{
+    /// <summary>
+    /// An action implementation which does nothing for filling type parameter taking of
+    /// <see cref="IAction"/>.
+    /// </summary>
+    public class NullAction : IAction
+    {
+        public NullAction()
+        {
+            PlainValue = default(Null);
```
Comment (dahlia) [inline Libplanet/Action/NullAction.cs:13]: Such expression is no more usable with null checker turned on.  Please use `Null.Value` instead.  See also <https://github.com/planetarium/bencodex.net/issues/20> for details.

```suggestion
            PlainValue = Null.Value;
```

File: Libplanet/Action/NullAction.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        }
+
+        public IValue PlainValue
+        {
+            get;
+            private set;
+        }
+
+        public void LoadPlainValue(
+            IValue plainValue)
```
Comment (dahlia) [inline Libplanet/Action/NullAction.cs:23]: Though it's trivial, but an unnecessary new line.

```suggestion
        public void LoadPlainValue(IValue plainValue)
```

File: Libplanet/Libplanet.csproj [hunk 5]
```diff
@@ -112,9 +109,7 @@ https://docs.libplanet.io/</Description>
   <!-- The above hacky trick is borrowed from the following Stack Overflow
   answer: https://stackoverflow.com/a/45004898/383405 (see #2). -->
   <ItemGroup>
-    <ProjectReference
-      Include="..\Libplanet.Stun\Libplanet.Stun.csproj"
-      PrivateAssets="All" />
```
Comment (dahlia) [inline Libplanet/Libplanet.csproj:117]: I believe all changes on this file seems unnecessary and even unintended.

**Review by dahlia — APPROVED** [review]

**Pull request merged**